### PR TITLE
Add missing classes of section tags

### PR DESCRIPTION
**Pull request by soodssr**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌
| Fixed issues  | -

Tag classes created for Body, Article, Section, Nav, Aside, Hgroup, Header, Footer and Address.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -18,6 +18,7 @@
   `Html::h5()`, `Html::h6()` (vjik)
 - New #100: Add classes for tags `Picture`, `Audio`, `Video`, `Source` and `Track` (Gerych1984, vjik)
 
+- New #89: Add method `nofollow()` to the `A` tag (soodssr)
```
Comment (vjik) [inline CHANGELOG.md:21]: This is artefact of merging. Need remove this line.

Can you add line about changes?

File: tests/common/Tag/NavTest.php [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+final class NavTest extends TestCase
+{
+    public function testBase(): void
+    {
+        $this->assertSame(
+            "<nav><ul>\n<li>Home</li>\n<li>About Us</li>\n<li>Contact Us</li>\n</ul></nav>",
+            (string) Nav::tag()->content(
+                Ul::tag()->items(
+                    Li::tag()->content('Home'), Li::tag()->content('About Us'),
+                    Li::tag()->content('Contact Us')
```
Comment (vjik) [inline tests/common/Tag/NavTest.php:21]:
```suggestion
                    Li::tag()->content('Home'),
                    Li::tag()->content('About Us'),
                    Li::tag()->content('Contact Us')
```

**Review by vjik — CHANGES_REQUESTED** [review]
For new tags need method in `Html` helper. For example, `Html::body()`.

File: CHANGELOG.md (hunk 1 above)
Comment (soodssr) [reply CHANGELOG.md:21]: sure

File: tests/common/Tag/NavTest.php (hunk 2 above)
Comment (soodssr) [reply tests/common/Tag/NavTest.php:21]: noted

**Review by vjik — APPROVED** [review]

**Pull request merged**